import/webview/WebView.qml

```diff
@@ -259,15 +259,60 @@ RawWebView {
             return null
         }
 
+        function hasWebViewPage() {
+            return (webview.webViewPage == null || webview.webViewPage == undefined)
```

Comment:
this check is the wrong way around, I think.  it should be: return (webview.webViewPage != null && webview.webViewPage != undefined)